ipa-core/src/ff/boolean_array.rs

```diff
@@ -262,12 +209,13 @@ macro_rules! impl_serializable_trait {
                 let min_value = $name::ZERO.0;
                 deserialize(min_value).unwrap();
 
-                let one = $name::ONE.0;
-                deserialize(one).unwrap();
-
                 let mut max_value = $name::ZERO.0;
                 max_value[..$bits].fill(true);
                 deserialize(max_value).unwrap();
+
+                let mut rnd_value = $name::ZERO.0;
```

Comment:
does it check something that [this](https://github.com/private-attribution/ipa/blob/7e9432f7bfebb30bb57c0b181e8445f369049873/ipa-core/src/ff/boolean_array.rs#L748) test doesn't already?

Reply:
No, I don't think it does. I added this one to replace the test case in this function that deserialized the ONE value, but I can drop it in favor of the other test.